docs/INSTALL.md

```diff
@@ -315,4 +315,4 @@ Adjust the "certbot" command below according to what domain names you want to re
 - If you change any of the TypeScript or CSS, then you will need to re-run the `build_client.sh` script in order to re-bundle it into `main.min.js` and `main.min.css`. (This step does not require a server restart, but you will need to perform a hard cache refresh in the browser.)
   - Alternatively, if you are actively changing or developing TypeScript code, leave the `webpack-dev-server.sh` script running and go to "http://localhost/?dev". That way, the code will be automatically compiled whenever you change a file, and the page will automatically refresh.
   - You can also go to "http://localhost/?dev&login=test1" to automatically log in as "test1", "http://localhost/?dev&login=test2" to automatically log in as "test2", and so forth. This is useful for testing a bunch of different users in tabs without having to use an incognito window.
-- If you change any CSS, you might also need to run `build_client.sh crit` to re-generate the critical CSS, which is necessary for the content the users see first. The "crit" version takes longer than `build_client.sh`, so you only need to run it once before committing your changes.
+- If you change any CSS, you might also need to run `build_client.sh crit` to re-generate the critical CSS, which is necessary for the content the users see first. The "crit" version takes longer than `build_client.sh`, so you only need to run it once before committing your changes.
```

Comment:
what did you change here?

Reply:
I copied it from current master as ti was

Reply:
well thats why im confused, if its the same then it shouldn't show up in the PR diff